conf/provisioning/dashboards/nuodb/disk.json

```diff
@@ -1980,8 +2271,7 @@
           "tags": []
         },
         {
-          "alias": "Write Throttling",
-          "dsType": "influxdb",
+          "alias": "$tag_db:writethrottle",
```

Comment:
Can we use the same alias as in the other panels? I.e `$tag_host(db: $tag_db, nodeid: $tag_nodeid, startid: $tag_startid)`.

Reply:
not really.  writethrottle is a measurement on the TE we just average all of them for the database here.   I'm only interested in some indication that write throttling is happening.  not details broken down by each TE.

Reply:
Is this going to print a red line for each TE or a single line per database?

Reply:
single line per database.  I actually changed recently to be a pretty translucent bar graph.  Not sure if this shows that way or not.  In general seeing red on this graph is an indication that the archiver is falling behind and can say dirty atoms quick enough (archive queue is too deep).